CppCoreGuidelines.md

```diff
@@ -20279,7 +20279,7 @@ There seem to be four main reasons given for not using exceptions:
 There is no way we can settle this issue to the satisfaction of everybody.
 After all, the discussions about exceptions have been going on for 40+ years.
 Some languages cannot be used without exceptions, but others do not support them.
-This leads to strong traditions for the use and non-use of exceptions, and to heated debates.
```

Comment:
"Strong" doesn't necessarily mean "established" -- there can be established traditions that are weak, and others people will fight for. So please revert this, or make it "strongly established" perhaps?

Reply:
just reverted.